### PR TITLE
Update env variable and remove unwanted code

### PR DESCRIPTION
**Pull request by shivamhelp**
Remove unwanted code and use environment variable instead of hard coded value.

### CODE REVIEW

File: bin/docker-execute [hunk 1]
```diff
@@ -259,10 +259,10 @@ for c in $(ssh -p 29418 -o "StrictHostKeyChecking no" jenkins@gerrit.helpshift.c
  do ssh -p 29418 -o "StrictHostKeyChecking no" jenkins@gerrit.helpshift.com gerrit review $c --abandon; \
 done;
 
-bin/dry-run.rb --provider gerrit $PACKAGE_NAME $REPO_NAME;
-rm -rf /home/dependabot/dependabot-core/tmp/$REPO_NAME'
+bin/dry-run.rb --provider gerrit $PACKAGE_NAME $REPO_NAME;'
+# rm -rf /home/dependabot/dependabot-core/tmp/$REPO_NAME'
```
Comment (ineham) [inline bin/docker-execute]: we can remove the commented out code

File: common/lib/dependabot/file_fetchers/base.rb [hunk 2]
```diff
@@ -792,28 +792,27 @@ def _clone_repo_contents(target_directory:)
...
-              <<~CMD
-              curl -Lo #{path}/.git/hooks/commit-msg https://errit.helpshift.com/tools/hooks/commit-msg
-             CMD
-           )
-           SharedHelpers.run_shell_command(
-              <<~CMD
-              chmod +x #{path}/.git/hooks/commit-msg
-             CMD
-           )
+          #   SharedHelpers.run_shell_command(
```
Comment (ineham) [inline common/lib/dependabot/file_fetchers/base.rb:801]: Same, remove the commented out code

**Review by ineham — APPROVED** [review]
rest LGTM

**Comment by shivamhelp** [comment]
I will remove the commented code in another commit

**Pull request merged**